src/main/java/org/jabref/logic/exporter/BibTeXMLExportFormat.java

```diff
@@ -20,22 +20,7 @@
 import javax.xml.datatype.XMLGregorianCalendar;
 import javax.xml.namespace.QName;
 
-import org.jabref.logic.importer.fileformat.bibtexml.Article;
```

Comment:
The problem are these removed imports (and similarly further down)

Reply:
i dont have org.jabref.logic.importer.fileformat.bibtexml folder